collections/_posts/2024-10-20-tabular-explainability.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: review
+title: "Multi-Layers attention-based explainability via transformers for tabular data"
+tags: transformer, explainability
+author: "Olivier Bernard"
+cite:
+    authors: "Andrea Trevino Gavito, Diego Klabjan, and Jean Utke"
+    title:   "Multi-Layers attention-based explainability via transformers for tabular data"
+    venue: "arXiv 2024"
+pdf: "https://arxiv.org/pdf/2302.14278"
+---
+
+# Notes
+
+* No code available :(
+
+
+# Highlights
+
+* Investigate explainable models based on transformers for tabular data
+* Use of knowledge distillation (master/student) to train a single head but multi-layers (blocs) transformer to facilitate explicability analysis
+* Propose a graph-oriented explainability method based on the set of single head attention matrices
+* Compare this approach to attention-, gradient-, and perturbation-based explainability methods
+
+&nbsp;
+
+# Introduction
+
+* The field of explainable Artificial Intelligence is named XAI and has received increasing interest over the past decade
+* XAI algorithms for DL can be organized into three major groups: perturbation-based, gradient-based, and, more recently, attention-based
+* Transformers posses a built-in capability to provide explanations for its results via the analysis of attention matrices
+
+<div style="text-align:center">
+<img src="/collections/images/tabular_explainability/tab_exp_1.jpg" width=700></div>
```

Comment:
Maybe correct `bloc` to `block` in the figure, but I would understand if you don't want to bother :wink:

Reply:
done